### ISSUE TITLE
Twitch.TV videos do not load and play.

### ISSUE THREAD

**Issue by cschwenz** [brave/browser-laptop#12398]
### Description

Twitch.TV videos fail to load and play (related: #1546, this is being reported separately because the original bug report came with no steps to reproduce).

### Steps to Reproduce

  1. Ensure shields are up, "HTTPS Everywhere", "Fingerprinting Protection", and "Block Phishing/Malware" are on.
  2. Turn on the blocking of all ads, but turn off all of the additional filter lists.
  3. Turn on the blocking of third party cookies.
  4. Go to https://www.twitch.tv/videos/209500225 (hereafter the example page).
  5. Notice there is a black box where the video should be.
  6. Allow ads and tracking, then reload the example page.
  7. Still a black box.
  8. Turn off "HTTPS Everywhere", then reload the example page.
  9. Still a black box.
  10. Allow all cookies, then reload the example page.
  11. Still a black box.
  12. Turn off "Fingerprinting Protection", then reload the example page.
  13. You should now see the video's placeholder image with a loading spinner overlaid.  This will be followed by an ad playing (if you've allowed twitch.tv to autoplay media).
  14. Once the ad is done, the actual video will be available to play (there will be a big play button in the middle of the placeholder image).
  15. Press the play button in the middle of the placeholder image.
  16. The placeholder image will flash and return you to the state it was in at the start of step 15.  I have not been able to find any configuration which will allow the video on the example page to play.

**Actual result:**

The video on the example page (https://www.twitch.tv/videos/209500225) fails to load and play.

**Expected result:**

The video on the example page (https://www.twitch.tv/videos/209500225) should load and play.

**Reproduces how often:**

100% (from the UK, unknown from anywhere else)

### Brave Version

**about:brave info:**

 ```
Brave: 0.19.122 
rev: 009e79278cba9e3e0e5498d64739b0cb0a5b4696 
Muon: 4.5.31 
libchromiumcontent: 63.0.3239.108 
V8: 6.3.292.48 
Node.js: 7.9.0 
Update Channel: Release 
OS Platform: macOS 
OS Release: 17.3.0 
OS Architecture: x64
```

**Reproducible on current live release:**

Yes.

### Additional Information

Sorry if the steps to reproduce sound snarky; that is not the intent.  There are simply a _lot_ of variables in play.  😿

**Comment by LaurenWags** [comment]
+1 from @alexwykoff and twitter: https://twitter.com/pincho_gaming/status/1007521644078944256

**Comment by NumDeP** [comment]
Maybe the commit in #14724 for 0.24.x will fix this issue as well.

**Comment by bsclifton** [comment]
Fixed with https://github.com/brave/muon/pull/629; this is a duplicate of https://github.com/brave/browser-laptop/issues/14475
